src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.unit.spec.js

```diff
@@ -0,0 +1,32 @@
+// Dependencies.
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+// Relative imports.
+import { App } from './index';
+
+describe('Prescriptions Refill Page <App>', () => {
+  it('renders what we expect', () => {
+    const wrapper = shallow(<App />);
+
+    const text = wrapper.text();
+    expect(text).to.include(
+      'How can the VA Prescription Refill and Tracking tool help me manage my',
+    );
+    expect(text).to.include('Am I eligible to use this tool?');
+    expect(text).to.include('Once I’m signed in, how do I get started?');
+    expect(text).to.include(
+      'Can I use this tool to refill and track all my VA prescriptions?',
+    );
+    expect(text).to.include('Where will VA send my prescriptions?');
+    expect(text).to.include(
+      'How long will my prescriptions take to arrive, and when should I',
+    );
+    expect(text).to.include(
+      'Will my personal health information be protected?',
+    );
+    expect(text).to.include('What if I have more questions?');
+
+    wrapper.unmount();
+  });
+});
```

Comment:
You could abstract this content into an `Unauth` component and then just test for that component from this test. I'm actually curious if Mocha allows this since there is a  SCSS import in App

Reply:
I'm not really sure what you mean by an `Unauth` component 🙂 Could you rephrase?

Reply:
Yeah I just mean chunking out all of the content out of App and into a single `UnauthenticatedContent` view so that -

```
if (featureEnabled === false) return <UnauthenticatedContent/>
```

Reply:
So I would do that if the unauthenticated content differed considerably from the authenticated content, but i would say around 80% of it is the same currently, so I'll defer this request until the content begins to diverge more if that's ok! 🙂